Paymetheus.Rpc/WalletClient.cs

```diff
@@ -643,5 +644,201 @@ public void CancelRequests()
                 return Tuple.Create(walletMutex, syncTask);
             }
         }
+
+        public async Task StartAutoBuyer(AutoBuyerProperties properties)
+        {
+            try
+            {
+                var client = new TicketBuyerService.TicketBuyerServiceClient(_channel);
+                var req = new StartAutoBuyerRequest();
+                req.Passphrase = ByteString.CopyFrom(properties.Passphrase);
+                req.Account = properties.Account;
+                req.BalanceToMaintain = properties.BalanceToMaintain;
+                req.MaxFeePerKb = properties.MaxFeePerKb;
+                req.MaxPriceRelative = properties.MaxPriceRelative;
+                req.MaxPriceAbsolute = properties.MaxPriceAbsolute;
+                req.VotingAddress = properties.VotingAddress;
+                req.PoolAddress = properties.PoolAddress;
+                req.PoolFees = properties.PoolFees;
+                req.MaxPerBlock = properties.MaxPerBlock;
+                await client.StartAutoBuyerAsync(req);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
```

Comment:
please never do this. the caller must handle the exception, we can't do that here.